release.sh

```diff
@@ -3,39 +3,98 @@
 set -e
 set -x
 
-if [ ! -d ${HOME}/google-cloud-sdk ]; then
-  wget https://dl.google.com/dl/cloudsdk/channels/rapid/downloads/google-cloud-sdk-206.0.0-linux-x86_64.tar.gz -O install.tar.gz
-  tar -xvzf install.tar.gz
+KEY_FILE=
+SPIN_GCS_BUCKET_PATH=
+VERSION=
+
+
+function process_args() {
+  while [[ $# > 0 ]]
+  do
+    local key="$1"
+    shift
+    case $key in
+      --gcs_bucket_path)
+        SPIN_GCS_BUCKET_PATH="$1"
+        shift
+        ;;
+      --key_file)
+        KEY_FILE="$1"
+        shift
+        ;;
+      --version)
+        VERSION="$1"
+        shift
+        ;;
+      --help|-help|-h)
+        print_usage
+        exit 13
+        ;;
+      *)
+        echo "ERROR: Unknown argument '$key'"
+        exit -1
+    esac
+  done
+}
+
+
+function print_usage() {
+    cat <<EOF
+usage: $0 [--gcs_bucket_path <path>]
+    [--key_file <key file path>] [--version <version>]
```

Comment:
key_file and version are required so shouldnt have [] around them. I'd put --gce_bucket_path last since it is optional